packages/application-shell/src/test-utils/custom-views-utils.tsx

```diff
@@ -23,8 +30,12 @@ const testCustomViewData: CustomViewData = {
 type TRenderCustomViewParams = {
   locale: string;
   projectKey?: string;
+  projectAllAppliedPermissions?: { name: string; value: boolean }[];
   customViewHostUrl?: string;
   customViewConfig?: Partial<CustomViewData>;
+  apolloClient?: ApolloClient<NormalizedCacheObject>;
+  environment?: Partial<TProviderProps<{}>['environment']>;
+  user?: Partial<TProviderProps<{}>['user']>;
```

Comment:
I'm also adding the `environment` and the `user` as they also seem quite helpful when testing Custom Views with different permissions.